rllib/agents/trainer.py

```diff
@@ -826,6 +828,97 @@ def compute_action(self,
         else:
             return result[0]  # backwards compatibility
 
+    def compute_actions(self,
```

Comment:
Could we also deprecate `compute_action` and rename it into `compute_single_action`. Just add a `deprecation_warning("compute_action", "compute_single_action")` call at the beginning of `compute_action` and then return the results of `compute_single_action` from within there.

Reply:
Actually, this is fine. We can do this quickly in a separate PR.